pulsar-client/src/main/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImpl.java

```diff
@@ -383,6 +383,12 @@ protected Message<T> internalReceive() throws PulsarClientException {
             unAckedMessageTracker.add(message.getMessageId(), message.getRedeliveryCount());
             resumeReceivingFromPausedConsumersIfNeeded();
             return beforeConsume(message);
+        } catch (IllegalArgumentException e) {
```

Comment:
It looks `checkState(message instanceof TopicMessageImpl);` will throw this exception.

When will this happen? Could you shared your case?